README.md

```diff
@@ -13,27 +13,30 @@ Beginning with the next version (0.9.0), **JWST requires Python 3.5 and above**.
 
```

Comment:
Now that we're past version 0.9.0, let's change the wording of line 12 to "Beginning with version 0.9.0, ..."